### ISSUE TITLE
gatsby build failed with `Can't resolve '../components/SideBar' in '/home/masai/acharya.io/src/pages'` message

### ISSUE THREAD

**Issue by codehia** [anubhavsrivastava/gatsby-starter-prologue#2]
I am getting the following error when trying `Gatsby develop` command, with the following error.

`

 ERROR #98123  WEBPACK

Generating development JavaScript bundle failed

Can't resolve '../components/SideBar' in '/home/masai/acharya.io/src/pages'

File: src/pages/404.js

 ERROR #98123  WEBPACK

Generating development JavaScript bundle failed

Can't resolve '../components/SideBar' in '/home/masai/acharya.io/src/pages'

File: src/pages/index.js

failed Re-building development bundle - 0.111s
`

**Comment by anubhavsrivastava** [comment]
Can you specify NPM/Yarn version and/or gatsby globally installed version? 

Build is working fine for me. But, I believe "works on my machine" is not the solution you are looking out for.

**Comment by codehia** [comment]
I have npm version 6.13 and
 Gatsby CLI version: 2.8.22

just a small observation, the Sidebar folder in src is imported as SideBar in pages. I am new to Gatsby, in fact this is my first time dealing with Gatsby in the first place. So any explanation regarding this would be  a huge benefit.

Thanks for the quick response.

**Comment by anubhavsrivastava** [comment]
I checked with multiple version and importing SideBar is not an issue with version of node.

The issue here is, as rightly pointed out by you, folder name. 
I am using mac (linux based FS OS) and probably, you are on windows machine.
I will try to check on windows machine as soon as possible.

Mean while can you just rename the folder to match the import, i.e change the folder name to "SideBar". 
Basically, the import statement requires SideBar/index.js (default file looked up if Folder is a path in import), but it cannot find the path "SideBar". This certainly has to do something with OS level.

**Comment by codehia** [comment]
Hi, I think the issue was in the os level because the name change actually works. Just a suggestion, can I raise a PR for the same to fix it?

**Comment by anubhavsrivastava** [comment]
Yes please, that would be nice!

**Comment by codehia** [comment]
fantastic, since I'm new to development, It'll take me a few hours to sort out the forking and PR process.
Thanks
